releasenotes/notes/dont-add-sitepackages-redundantly-b0b45bdd81c5d361.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    No need to add site-packages twice to the sys.path, its misleading and
+    futile. It already took precedence over other locations.
```

Comment:
not sure if a user would understand this release note entry 🤔 

maybe `The checks packaged with the new wheels method are in the python ``sys.path`` by default, so we removed this path them from the locations that are appended to the default python ``sys.path`` `? Let me know